### PR TITLE
Revamp workshop scedule and reorder workshop list in workshops/README.md

### PR DESCRIPTION
**Pull request by CoolBeans19**
Changed core workshop schedule to reflect the updated plan. Also organized workshop list alphabetically.

### CODE REVIEW

**Comment by CoolBeans19** [comment]
pls give feedback and suggestions

File: workshops/README.md [hunk 1]
```diff
@@ -14,43 +16,49 @@ Likewise, if you have any questions or run into any trouble, please reach out to
 
 We recommend you run through these workshops first. They are built with total beginners in mind, so don't worry if you haven't coded before.
 
-You'll start off by building your first website and will eventually make your way through the fundamentals of game development.
+You'll start off by building your first website and will eventually make your way through the fundamentals of databases.
```
Comment (MaxWofford) [inline workshops/README.md]: Did you mean to put `databases`?

File: workshops/README.md [hunk 2]
```diff
@@ -14,43 +16,49 @@ Likewise, if you have any questions or run into any trouble, please reach out to
...
-4. [Soccer][soccer]: _Build your first browser game._
-5. [Dodge][dodge]: _Expand upon what you started in Soccer and build your second, more advanced game._
-6. [Maze][maze]: _This one's super free-form. Build your third game and thoroughly customize it to truly make it your own._
-7. [Ajar.io][ajar]: _Much more advanced and detailed than previous game workshops. Build a single-player clone of https://agar.io/._
+3. [That Was Easy][that_was_easy]: _Create a website with interactive elements._
+4. [Geometric Pattern][geometric_pattern]: _Create rad shapes and patterns through code._
+5. [Dodge][dodge]: _Build your first browser game._
+6. [Platformer][platformer]: _Build on what you learned in Dodge to create your second, more complex game._
+7. [Chat][chat]: _Use a database to create a live chat website._
+8. [Collaborative Sketch][collaborative_sketch]: _Use your new database skills to create a real-time collaborative drawing app_
```
Comment (MaxWofford) [inline workshops/README.md]: This should end with a period.

**Comment by CoolBeans19** [comment]
Made changes @MaxWofford suggested, want to give any more feedback @zachlatta?

**Comment by CoolBeans19** [comment]
I'm going to merge at 2 if I don't get any more suggestions @MaxWofford @zachlatta

**Comment by MaxWofford** [comment]
Looks good to me!

partyparrot

**Pull request merged**